src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -287,6 +187,73 @@
       },
       "required": [ "value", "algorithm" ]
     },
+    "invocation": {
+      "description": "The runtime environment of the analysis tool run.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+
+        "parameters": {
+          "description": "A string containing the runtime parameters with which the tool was invoked. For command line tools, this string may consist of the completely specified command line used to invoke the tool.",
+          "type": "string"
+        },
+
+        "startTime": {
+          "description": "The date and time at which the run started. See \"Date/time properties\" in the SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "endTime": {
+          "description": "The date and time at which the run ended. See \"Date/time properties\" in the  SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "correlationId": {
+          "description": "An identifier that allows the run to be correlated with other artifacts produced by a larger automation process.",
+          "type": "string"
+        },
+
+        "machine": {
```

Comment:
I didn't even notice "machine" in the original proposal, but... if we consider something important enough to be a first-class member of `invocation`, the decision shouldn't depend on whether that thing happens to exist as an environment variable on a particular operating system.